cookbook/doctrine/registration_form.rst

```diff
@@ -5,28 +5,36 @@
 How to Implement a simple Registration Form
 ===========================================
 
-Some forms have extra fields whose values don't need to be stored in the
-database. For example, you may want to create a registration form with some
-extra fields (like a "terms accepted" checkbox field) and embed the form
-that actually stores the account information.
+Creating a registration form is pretty easy - it *really* means just creating
+a form that will update some ``User`` object (a Doctrine entity in this example)
+and then save it.
 
-The simple User Model
----------------------
+If you don't already have a ``User`` entity and a working login system,
+first start with :doc:`/cookbook/security/entity_provider`.
 
-You have a simple ``User`` entity mapped to the database::
+Your ``User`` entity will probably at least have the following fields:
+* ``email``
+* ``username``
```

Comment:
At the end of the article there is a proposed section about registration without usernames. It would be nice to explain here what a `username` can be. Is only the traditional login? Can be an email? Can I use other property as a username (in the `security.yml` config file I can configure the username)?